### PR TITLE
refactor format.js

### PR DESCRIPTION
**Pull request by pycoinfu**

### CODE REVIEW

File: guide/scripts/format.js [hunk 1]
```diff
@@ -11,35 +11,37 @@ function toc() {
     for (let h of headers) {
         title = h.textContent;
         cls = h.getAttribute("id");
-        h.href = `#${cls}`;
+        h.href = '#' + cls;
         html += '<li><a href="#' + cls + '">' + title + '</a></li>';
     }
 
-    table_of_contents.innerHTML = html;
+    tablOfContents.innerHTML = html;
```
Comment (SSS-Says-Snek) [inline guide/scripts/format.js:18]: Typo

**Review by SSS-Says-Snek — CHANGES_REQUESTED** [review]

File: guide/scripts/format.js (hunk 1 above)
Comment (pycoinfu) [reply guide/scripts/format.js:18]: Thanks for pointing that out!

**Review by SSS-Says-Snek — APPROVED** [review]
LGTM!

**Review by oddbookworm — APPROVED** [review]
LGTM

**Pull request merged**